src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -110,7 +110,7 @@ const noticeCss = {
     border: '1px solid #cacaae',
     padding: 3,
     borderRadius: 2,
-    marginBottom: 3,
+    marginBottom: 10,
```

Comment:
Let's add this to the style.  It'll look better when `Workspace` is selected.
```
    white-space: nowrap;
    text-align: center;

```

Reply:
Done, thanks!